src/analysis/calculus/cont_diff.lean

```diff
@@ -1580,11 +1580,15 @@ begin
     refl }
 end
 
+@[simp] lemma iterated_fderiv_zero_fun_apply {n : ℕ} {x : E} :
```

Comment:
If you make `iterated_fderiv_zero_fun` into a `simp` lemma, then `simp` can automatically prove this one.